cmd/oci/oci.go

```diff
@@ -22,8 +22,8 @@ import (
 )
 
 var (
-	platform, output             string
-	push, loadDocker, loadPodman bool
+	platform, output                    string
+	push, loadDocker, loadPodman, login bool
```

Comment:
login shouldn't be a flag be a dedicated command. Like `bsf oci login`.

Reply:
ok